test/test_hdf5_parallel.py

```diff
@@ -0,0 +1,54 @@
+# Unit and other tests
+import datetime
+import gdal
+import math
+import numpy as np
+import os
+import pandas as pd
+import unittest
+import scipy.io as sio
+import subprocess
+
+
+class hdf5_parallel_Tests(unittest.TestCase):
+
+    stepSize = 15.0
+    pnts_file = '../test/scenario_1/geom/query_points.h5'
+    wm_file = '../test/scenario_1/weather_files/ERA5_2020-01-03 23:00:00_15.75N_18.25N_-103.25E_-99.75E.h5'
+    interpType = 'rgi'
+    verbose = True
+    delayType = ['Zenith']
+    
+    from RAiDER.delayFcns import get_delays
+    
+    delays_wet_1, delays_hydro_1 = get_delays(stepSize, pnts_file, wm_file, interpType,
+                                              verbose, delayType, cpu_num = 1)
+    conts = sio.loadmat('get_delays_time_elapse.mat')
+    time_elapse_1 = conts['time_elapse'][0]
+    delays_wet_4, delays_hydro_4 = get_delays(stepSize, pnts_file, wm_file, interpType,
+                                              verbose, delayType, cpu_num = 4)
+    conts = sio.loadmat('get_delays_time_elapse.mat')
+    time_elapse_4 = conts['time_elapse'][0]
+    
+    subprocess.getoutput('rm get_delays_time_elapse.mat')
+
+    # test error messaging
+    def test_get_delays_wet_accuracy(self):
+        self.assertTrue(np.allclose(self.delays_wet_1, self.delays_wet_4))
+
+    
+    def test_get_delays_hydro_accuracy(self):
+        self.assertTrue(np.allclose(self.delays_hydro_1, self.delays_hydro_4))
+
+    
+    def test_get_delays_runtime(self):
```

Comment:
I'm thinking that instead of testing for the parallel time being less than the serial time, we simply print out the speedup.

Reply:
For now, each time get_delays is executed, the runtime is printed out. So for this unit test, the runtimes for both serial and parallel time have already been printed out. Do you still want to print out the speedup?

Reply:
I think we can put the time printout in the get_delay fcn under the verbose option (or logging once we get it). If we do that, then we can just delete this unit test. I'm unsure whether it's a good idea to have a unit test fail if there is no speedup, since we don't know all possible system setups.